### DISCUSSION TITLE
Using smart-autocliker not only with image recognition as a condition.

### DISCUSSION THREAD

**Discussion by avatitsi** [Nain57/Smart-AutoClicker#256] [Q&A]
I try to learn to use the smart autoclicker in depth. But i came across with one problem . If i need to make actions i always have to fulfill a condition with an image capture. Is that so? If i want to make a few random clicks without any image recognition first, isn't that possible?
I do something wrong? Also on this app the conditions are only image recognition procedures? There is no option a condition to be a counter for example?

**Comment by hatbrox** [comment]
I would also like to have a timer (10, 9, 8... action) but the problem is that you still need an event to trigger the timer.
the device screen is the only thing the app can visualise to test conditions.
workaround is to have an image dummy recognition (something that will always be true) and then use the Wait action type before you trigger another action.
maybe have a timer that starts immediately after you press the play button when you launch a scenario.
